lte/gateway/python/integ_tests/defs.mk

```diff
@@ -13,7 +13,8 @@
 PROTO_LIST:=orc8r_protos lte_protos feg_protos
 
 # Add the s1aptester integration tests
-PRECOMMIT_TESTS = s1aptests/test_attach_detach.py \
+PRECOMMIT_TESTS = s1aptests/test_services_are_running.py \
```

Comment:
For my understanding, how this testcase is actually benefitting? This testcase checks the services status, prints the same and if the status is not well, it asserts with failure, but continues with rest of the testcases even if there is a failure for some of the services. Are we planning to restrict rest of the testcases to execute in future, if services are not running as per this testcase check?

Reply:
Motivation/benefit of this test:
* In #14007 enodebd was refactored, but the newly added folders were not added to the setup.py. This is, in the magma.deb created by the build all job, enodebd was damaged and in a restart-loop.
* This did not make the integration tests vs the artifact failing (https://github.com/magma/magma/actions/workflows/lte-integ-test-magma-deb.yml). This was discovered by chance.
* This test should prevent that we create a magma.deb with a damaged enodebs service without noticing.

It did not seem right to create a test that only checks for enodebd - this is why we created a test that checks for all services. Yes, for a lot of services this is redundant, because a lot of tests (if not even all) will already fail if, e.g., mme or pipelined i not running.

There is not plan yet to restrict other test runs if a service is not running - I do not know if it would be a good idea to make unrelated tests fail if e.g., enodebd is not running. I would think for tests that need certain services it is sufficient if they fail on their own. What do you think?

Reply:
Thank you for the information @nstng Makes sense.